README.md

```
@@ -1,3 +1,5 @@
+GHA TEST 1
```

markdownlint-cli2 (0.20.0)

[warning] First line in a file should be a top-level heading

(MD041, first-line-heading, first-line-h1)